### PR TITLE
Simplify rule loading based on specific file rather than scanning fol…

### PR DESCRIPTION
**Pull request by objectiser**
…der, as became very complex when attempting to scan nested jars within spring boot uber jar.

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkrohling @pavolloffay Could you review please?

**Comment by pavolloffay** [comment]
Does it mean that all rules from one source (which I guess is jar) have to be in one file?

**Comment by objectiser** [comment]
@pavolloffay yes - but that is probably always going to be the case anyway.

File: rules/java-web-servlet-filter/src/main/resources/otarules.btm [hunk 1]
```diff
@@ -20,4 +20,22 @@ DO
   setState($0.getServletContext(), 1);
 ENDRULE
 
+RULE Tomcat servlet filter installation
```
Comment (pavolloffay) [inline rules/java-web-servlet-filter/src/main/resources/otarules.btm:23]: nit: `otrarules.btm` in other rule-modules, the name is based on technology/framework.

**Comment by pavolloffay** [comment]
LGTM, once those files become long we might solve this differently.

**Comment by objectiser** [comment]
@pavolloffay Thanks! Hopefully the files shouldn't get too long as only installing the framework integrations.

File: rules/java-web-servlet-filter/src/main/resources/otarules.btm (hunk 1 above)
Comment (pavolloffay) [reply rules/java-web-servlet-filter/src/main/resources/otarules.btm:23]: sorry, I did not see that all rule files are renamed to `otarules.btm`.

**Pull request merged**